guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -4,37 +4,64 @@
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
+.Subscriptions service capabilities
+The Subscriptions service is a service that helps organizations track and manage their Red{nbsp}Hat subscription usage across connected and disconnected environments. 
+* Allows customers to view subscription usage.
+* When configured, it displays usage data from both connected and disconnected environments.
+* Tracks Red{nbsp}Hat product usage and capacity at the fleet or account level.
+* Generates a daily snapshot of usage data across all tracked systems.
+* Displays data in a filterable dashboard at cloud.redhat.com.
+* Collects historical data for analytics, purchasing decisions, capacity planning, and risk mitigation.
+* Supports the transition to Simple Content Access (SCA), which enforces subscriptions across the organization rather than on individual systems.
+
+.Configuring your system to use the Subscriptions service
+ifndef::satellite[]
 .Prerequisites
-* Ensure you have a Red{nbsp}Hat subscription manifest.
-ifdef::orcharhino[]
-You can create and download your manifest in the https://access.redhat.com/management/subscription_allocations[Red Hat Customer Portal].
-endif::[]
-ifndef::orcharhino[]
-** If your {Project} is connected, use the {RHCloud} to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
-endif::[]
-** If your {Project} is disconnected, use the Red Hat Customer Portal to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
-endif::[]
+* Enable the `foreman_rh_cloud` plugin:
+[subs="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# foreman-installer --enable-foreman-plugin-rh-cloud
+----
+
+Connected {Project} Users::
+Connected {Project} Users should configure the `foreman_rh_cloud` plugin to share inventory information with the Hybrid Cloud Console.
+Ensure the *Automatic Inventory Upload* setting is enabled.
+The plugin enables the usage of the subscriptions service to track usage information across connected systems.
+
+The plugin can be configured to omit data that is not pertinent for subscription purposes, such as host names and IP addresses.
 endif::[]
 
+Disconnected users::
+Users who cannot or do not want to connect directly can export a local summary of their product usage and import it into a CMDB, spreadsheet, or other tracking tools.
+
+Disconnected ({Project})::
+{Project} users can export usage data in one of two ways:
```

Comment:
These parts aren't user-focused enough. Instead of speaking about "users", speak to the user directly.